Enemies/Enemy.gd

```diff
@@ -282,3 +311,8 @@ func alert_to_player() -> void:
 
 func update_last_player_position(position: Vector3) -> void:
 	last_player_position = position
+
+func set_damage(d_mult: float):
```

Comment:
What are these functions doing?

Reply:
Give functionality to raise or lower the amount of danger an enemy is doing/revert the changed damage to default. Was made in an effort to stop crouching enemies from raising danger because that is just not fair. I'm not sure if it solved the problem but I liked the idea of being able to change it with certain actions. For example a sniper enemy does no danger until it is in an "aiming" action or something

Reply:
Ah, were enemies raising danger while they were completely hidden from the player? I agree it may be time to give enemies more flexibility in how they endanger the player, that code in general is due for a bit of modification.

Reply:
Yes, enemies were causing danger while crouched and effectively hidden. 
Another improvement to the danger code would be a way to adjust the specific ranges, as right now the shotgun doesn't really have a way of doing less damage from further away but more close up, outside of manually setting the danger multiplier on range changes.